tests/getdos_TestCase.py

```diff
@@ -23,50 +23,56 @@
 
 class TestCase(unittest.TestCase):
```

Comment:
An inherited class using the parent's name is pretty confusing. We should look for this sort of craziness later and change it.

Reply:
agreed!!